packages/wallet-sdk/src/connection/RxWebSocket.ts

```diff
@@ -1,27 +1,29 @@
 // Copyright (c) 2018-2023 Coinbase, Inc. <https://www.coinbase.com/>
 // Licensed under the Apache License, version 2.0
 
-import { BehaviorSubject, empty, Observable, of, Subject, throwError } from 'rxjs';
-import { flatMap, take } from 'rxjs/operators';
+import { ServerMessage } from './ServerMessage';
```

Comment:
nit: should we rename this file to be `WebSocket` since we're no longer using RX?

Reply:
https://github.com/coinbase/coinbase-wallet-sdk/pull/1018#discussion_r1346250503
@cb-jake yup. but then github considered it as a new file and made it hard to compare from the previous implementation. will rename the file as followup.

Reply:
that makes sense